github/ci/prow/files/config.yaml

```diff
@@ -353,6 +353,23 @@ presets:
   - name: docker-config
     mountPath: /etc/default
     readOnly: true
+- labels:
+    preset-docker-mirror-proxy: "true"
+  env:
+  - name: DOCKER_HTTP_PROXY
+    value: http://docker-mirror-proxy.kubevirt-prow.svc:3128
+  - name: DOCKER_HTTPS_PROXY
+    value: http://docker-mirror-proxy.kubevirt-prow.svc:3128
+  - name: CA_CERT_FILE
+    value: /etc/docker-mirror-proxy/ca.crt
+  volumes:
+  - name: docker-mirror-proxy-ca-cert
+    secret:
+      secretName: docker-mirror-proxy
```

Comment:
This would probably have to be the configmap now.

Reply:
Makes total sense, done.